src/main/java/nl/hsac/fitnesse/fixture/util/selenium/driverfactory/DriverManager.java

```diff
@@ -69,7 +73,7 @@ public void setSeleniumHelper(SeleniumHelper helper) {
         this.helper = helper;
     }
 
-    protected SeleniumHelper createHelper(WebDriver driver) {
+    protected SeleniumHelper createHelper() {
```

Comment:
The driver was passed to this method so it could be used by subclasses (specifically the `nl.hsac.fitnesse.fixture.util.mobile.AppiumDriverManager` uses it to determine what type of device is used in tests. Please leave it in place.

Reply:
Put it back in. I will say this implementation has a smell; but since it's irrelevant for this pr, let's leave it at that.